lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -335,7 +336,12 @@ def stop_line_select
 
         def read_char
```

Comment:
It looks like you're using `@supports_arrow_keys` as a proxy for "is this windows?" instead of asking the question you really want to ask.  As I dig into the difference between `getc` and `getch`, it looks like `getch` forces raw mode, which is what `raw_tty!` should do, but seems to be failing to do on Windows.  `raw_tty!` is a private method being used only for this, so I'd recommend refactoring by removing it and doing:
```
        def read_char
          if $stdin.tty? && !ENV['TEST']
            $stdin.getch # raw mode for tty
          else
            $stdin.getc
          end
        rescue IOError
          "\e"
        end
```